client/signup/config/steps-pure.js

```diff
@@ -555,18 +555,16 @@ export function generateSteps( {
 			stepName: 'site-or-domain',
 			props: {
 				getHeaderText( domainCart ) {
-					return i18n.getLocaleSlug() === 'en' ||
-						i18n.hasTranslation( 'Choose how to use your domains' )
-						? i18n.translate( 'Choose how to use your domain', 'Choose how to use your domains', {
```

Comment:
Huh

Reply:
yeah. conditional plural or something. Amazing how much we leave lingering this way